fftools/ffmpeg_qsv.c

```diff
@@ -48,6 +48,21 @@ static int qsv_device_init(InputStream *ist)
 {
     int err;
     AVDictionary *dict = NULL;
+    HWDevice *hw_device = NULL;
+
+    if (ist->hwaccel_device) {
+        hw_device = hw_device_get_by_name(ist->hwaccel_device);
+        if (hw_device && hw_device->type == AV_HWDEVICE_TYPE_QSV) {
+            hw_device_ctx = av_buffer_ref(hw_device->device_ref);
```

Comment:
who will release this?

Reply:
Add unref operation in qsv_uninit(). It miss an unref even without this patchset. Seems it is a bug for ffmpeg-qsv.